Source/WebCore/PAL/pal/PALSwift/CryptoKitShim.swift

```diff
@@ -476,6 +476,116 @@ public enum EdKeyAgreementAlgorithm {
 }
 
 public class EdKey {
+    public static func generatePrivateKey(algo: EdSigningAlgorithm) -> VectorUInt8 {
+        switch algo {
+        case .ed25519:
+            return Curve25519.Signing.PrivateKey().rawRepresentation.copyToVectorUInt8()
+        case .ed448:
+            return Data(count: 0).copyToVectorUInt8()
+        }
+    }
+
+    public static func generatePrivateKeyKeyAgreement(algo: EdKeyAgreementAlgorithm) -> VectorUInt8
+    {
+        switch algo {
+        case .x25519:
+            return Curve25519.KeyAgreement.PrivateKey().rawRepresentation.copyToVectorUInt8()
+        case .x448:
+            return Data(count: 0).copyToVectorUInt8()
+        }
+    }
+
+    public static func privateToPublic(algo: EdSigningAlgorithm, priv: SpanConstUInt8)
```

Comment:
Nit:  What is `priv` short for?  If `privateKey`, I'd suggest spelling it out.

Nit  Similarly, `algorithm` is better than `algo`.

If these are already used elsewhere in the file, ignore this feedback.

Reply:
Let me do a follow up with long names. I can just replace all of such instances with longer/specific names.